code/game/objects/items/mop.dm

```diff
@@ -42,6 +43,68 @@
 	reagents.remove_any(val2remove) //reaction() doesn't use up the reagents
 
 
+/obj/item/mop/examine(mob/user)
+	. = ..()
+	. += "It has [!notches ? "nothing" : "[notches] notches"] scratched into the handle."
+
+/obj/item/mop/attack(mob/living/target, mob/living/user)
+	var/living_state = target.stat
+	. = ..()
+	if(!QDELETED(target) && target.stat == DEAD && target.mind && living_state != DEAD)
+		add_notch(user)
+
+/obj/item/mop/proc/add_notch(mob/living/user) //DYNAMIC CLAYMORE PROGRESSION SYSTEM - THIS IS THE FUTURE
+	notches++
+	force++
```

Comment:
uh oh.

fair warning, if you and a buddy got together to farm this up with repeated defibbings of one guy, you could easily get a mop to force 100+ in a couple of minutes. if you want to keep this force escalation part, you're probably gonna need to make mops keep a track of a list of minds (or ckeys?) that they have reaped so that nobody can be counted twice.